### PR TITLE
Reporter UI tweaks

### LINKED ISSUE #7803: Test Runner UI Tweaks [TR-43]
**Acceptance Criteria**
- Change “TEST” to “TEST BODY”
- When hovering a test, the dotted left border color changes to `#828282`
- When hovering a test, only the accordion/test header triggers the hover state
- Only clicking the accordion/test header triggers expand/collapse of accordion
- When hovering line items in a test, the text color is `#6A6B6C`

Design of hover:
Screen Shot 2020-06-24 at 6 07 11 PM

### PR DESCRIPTION
**Pull request by panzarino**
- Closes #7803 

### User facing changelog

Minor updates to improve the user experience in the reporter, full list of changes in linked issue

### How has the user experience changed?

Before:

ezgif-6-74c1da8fcb24

After:

ezgif-6-b67a0a5e99c1

### PR Tasks

- [ ] Have tests been added/updated?
- [x] Has the original issue been tagged with a release in ZenHub? 
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)?

### CODE REVIEW

File: packages/reporter/src/test/test.tsx [hunk 1]
```diff
@@ -71,48 +81,45 @@ class Test extends Component<Props> {
...
     return (
-      <div
-        ref='container'
-        className={cs('runnable-wrapper', { 'is-open': this._shouldBeOpen() })}
-        onClick={this._toggleOpen}
-        style={{ paddingLeft: indent(model.level) }}
+      <Collapsible
+        containerRef={this.containerRef}
+        header={this._header()}
+        headerClass='runnable-wrapper'
```
Comment (chrisbreiding) [inline packages/reporter/src/test/test.tsx:87]: I don't think `runnable-wrapper` should be the header class. It's meant to wrap the entire runnable. You could achieve this by updating the `<Collapsible />` component to accept a `className` prop, and then add that in to the root-level div's className. See my other code comment for further clarification.

That should fix the failing e2e tests. They rely on the `runnable-wrapper` class being on the top-most element of the runnable. It should also reduce the need for the changes to selectors in the `scss` file

File: packages/reporter/src/collapsible/collapsible.tsx [hunk 2]
```diff
@@ -38,7 +40,7 @@ class Collapsible extends Component<Props, State> {
 
   render () {
     return (
-      <div className={cs('collapsible', { 'is-open': this.state.isOpen })}>
+      <div className={cs('collapsible', { 'is-open': this.state.isOpen })} ref={this.props.containerRef}>
```
Comment (chrisbreiding) [inline packages/reporter/src/collapsible/collapsible.tsx:43]:
```suggestion
      <div className={cs('collapsible', this.props.className, { 'is-open': this.state.isOpen })} ref={this.props.containerRef}>
```

File: packages/reporter/src/collapsible/collapsible.tsx [hunk 3]
```diff
@@ -9,7 +9,9 @@ interface Props {
   headerStyle?: CSSProperties
   header?: ReactNode
   headerExtras?: ReactNode
+  containerRef?: RefObject<HTMLDivElement>
   contentClass?: string
+  toggleOpen?: (isOpen: boolean) => any
```
Comment (chrisbreiding) [inline packages/reporter/src/collapsible/collapsible.tsx:14]:
```suggestion
  toggleOpen?: (isOpen: boolean) => any
  className?: string
```

**Comment by panzarino** [comment]
@chrisbreiding so that's how it used to work because there was the custom collapsible system, however I feel like leaving it as the header class is more consistent with what is already being done with [`suite`](https://github.com/cypress-io/cypress/blob/develop/packages/reporter/src/runnables/runnable-and-suite.tsx#L25). I don't think that it makes sense to use the class differently in different contexts.

**Comment by chrisbreiding** [comment]
Makes sense. I didn't realize/remember that suites worked that way. In that case, I think you should be able to get the e2e tests passing by updating [this selector](https://github.com/cypress-io/cypress/blob/develop/packages/server/test/support/fixtures/projects/e2e/cypress/support/util.js#L48) to `.runnable-instruments`.

**Pull request merged**